requirements.pip

```diff
@@ -1,50 +1,48 @@
 # Krux' python repo
 --extra-index-url https://staticfiles.krxd.net/foss/pypi/
 
-### Krux' standard library, which this is built on
+# Krux' standard library, which this is built on
 krux-stdlib==2.1.1
 
-boto==2.38.0
+# The library this is wrapping
+boto==2.39.0
 
-###
-### Krux Stdlib's requirement.pip contents:
-###
-### XXX this is needed so there are no version conflicts when
-### one downstream library does NOT specify the version it wants,
-### and another one does.
-### Every time the version of krux-stdlib is updated, the list
-### below should be updated as well.
+# Transient libraries
+# This is needed so there are no version conflicts when
+# one downstream library does NOT specify the version it wants,
+# and another one does.
 
-# Simple templating engine
+# From krux-stdlib
 pystache==0.5.4
-
-# sphinx (for API documentation)
-Sphinx==1.2b1
-# needed by sphinx
-Jinja2==2.6
-# needed by sphinx
-Pygments==1.6
-docutils==0.10
-
-# stats
-kruxstatsd==0.2.2
+Sphinx==1.3.5
+Jinja2==2.8
+Pygments==2.1
+docutils==0.12
+kruxstatsd==0.2.4
 statsd==2.0.3
-
-# Latest tested versions
-argparse==1.2.1
-GitPython==0.3.2.RC1
-simplejson==3.3.0
-tornado==3.0.1
-# for lockfiles in cli
-lockfile==0.9.1
-
-# Test libraries
-coverage==3.6
-mock==1.0.1
-nose==1.2.1
-
-## The following requirements were added by pip --freeze:
-async==0.6.1
-fudge==1.0.3
-gitdb==0.5.4
-smmap==0.8.2
+argparse==1.4.0
+GitPython==1.0.2
+simplejson==3.8.1
+tornado==4.3
+lockfile==0.12.2
+async==0.6.2
+fudge==1.1.0
+gitdb==0.6.4
+smmap==0.9.0
+
+# From pip --freeze
+Babel==2.2.0
+MarkupSafe==0.23
+alabaster==0.7.7
+backports-abc==0.4
+backports.ssl-match-hostname==3.5.0.1
+certifi==2015.11.20.1
+pbr==1.8.1
+pygerduty==0.32.1
+pytz==2015.7
+singledispatch==3.4.0.3
+six==1.10.0
+snowballstemmer==1.2.1
+sphinx-rtd-theme==0.1.9
+wheel==0.24.0
+wsgiref==0.1.2
```

Comment:
Comments relating to why these versions were chosen/updated would be helpful in case they need to be updated in the future.  Will someone looking at this list know whether or not updating will break things?